badoo_selenium.py

```diff
@@ -9,24 +9,39 @@
     "message_intro_1": "Hi, are you in Kyiv?",
     "message_2": ("Great, let's chat on Instagram instead, @ig_handle_here"),
     "positive_words": ["yes"],
-    "username": "your_username",
-    "password": "your_password",
-    "webdriver_path": "path/to/your/webdriver",
+    "webdriver_path": "/Users/00-facebook/Downloads/chromedriver_mac64/chromedriver",
+    "facebook_email": "your_facebook_email",
+    "facebook_password": "your_facebook_password",
```

Comment:
Login with Badoo user/password is a requirement, not facebook login. Badoo login will work fine

Reply:
You were provided with a script previously that demos 100% working badoo user/pass login, use this for reference.